src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildACppCliProject.cs

```diff
@@ -40,7 +45,7 @@ public void It_builds_and_runs()
             var exe = Path.Combine( //find the platform directory
                 new DirectoryInfo(Path.Combine(testAsset.TestRoot, "CSConsoleApp", "bin")).GetDirectories().Single().FullName,
                 "Debug",
-                "net5.0",
+                ToolsetInfo.CurrentTargetFramework,
```

Comment:
This should either be hardcoded at `net7.0` to match the test asset, or the test asset should be updated to use CurrentTargetFramework.